src/pip/_internal/commands/install.py

```diff
@@ -529,14 +529,16 @@ def _warn_about_conflicts(self, conflict_details, resolver_variant):
         parts = []  # type: List[str]
         if resolver_variant == "legacy":
             parts.append(
-                "After October 2020 you may experience errors when installing "
-                "or updating packages. This is because pip will change the "
-                "way that it resolves dependency conflicts.\n"
+                "pip's legacy dependency resolver does not consider dependency "
+                "conflicts when selecting packages. This behaviour is the "
```

Comment:
Kind of OT, dwe have style guides on what spelling to use in error messages?

Reply:
Not really. It's a flip-a-coin-and-pick-one spelling approach right now.

As for the exact wording, that's getting iterated on in the issue, so it's mostly a "does everyone agree it's good enough". :P